### PR TITLE
Update OWNERS

### PR DESCRIPTION
**Pull request by cblecker**
Added a bunch of sub-OWNERS files too for auto review assignments

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
 
 aliases:
-  sre-platform-leads:
+  srep-functional-leads:
     - c-e-brumm
     - cblecker
     - jaybeeunix
+    - lisa
+    - NautiluX
+    - rogbas
```
Comment (wanghaoran1988) [inline OWNERS_ALIASES:10]: could you also add APAC function lead @dofinn and me as a backup?

File: OWNERS [hunk 2]
```diff
@@ -1,14 +1,6 @@
...
-- cblecker
-- jewzaam
-- mwoodson
-- wanghaoran1988
-- dofinn
+- srep-functional-leads
+- srep-team-leads
+approvers:
+- srep-functional-leads
+- srep-team-leads
```
Comment (jewzaam) [inline OWNERS:6]:
```suggestion
- srep-team-leads
maintainers:
- jewzaam
- mwoodson
- jharrington22
```

File: OWNERS_ALIASES [hunk 3]
```diff
@@ -1,12 +1,14 @@
 # See the OWNERS_ALIASES docs: https://git.k8s.io/community/contributors/guide/owners.md#OWNERS_ALIASES
 
 aliases:
-  sre-platform-leads:
+  srep-functional-leads:
     - c-e-brumm
```
Comment (jewzaam) [inline OWNERS_ALIASES:5]:
```suggestion
    - dofinn
```

**Comment by jewzaam** [comment]
/hold cancel
ready to look at merging, added some suggestions.  hopefully `maintainers` won't break things, it will be nice to track that in one place with reviewers and approvers

**Pull request merged**